lib/sanity-ruby.rb

```diff
@@ -0,0 +1,3 @@
+# frozen_string_literal: true
+
+require 'sanity'
```

Comment:
Running `bin/standard` here will catch the double-quoted strings preference

```suggestion
require "sanity"
```